pyproject.toml

```diff
@@ -25,8 +25,8 @@ requires-python = ">=3.8"
 dependencies = [
     "scipy",
     "numpy",
-    "pandas~=1.5.0",
-    "matplotlib>=3.5.0",
+    "pandas~=2.0.0",
+    "matplotlib>=3.6.1",
```

Comment:
In the interests of completeness, I think 3.6.1 is required by pandas 2.0.0, so you can drop this hard-coded number.

Reply:
Without it, it breaks the minimum requirements workflow.

Reply:
Unless you mean remove matplotlib completely?

Reply:
Ah -- you're right. pandas doesn't require matplotlib, but if it's present requires it at this level, so we do indeed need to specify.